src/v/cluster/topic_table.cc

```diff
@@ -161,10 +161,13 @@ topic_table::apply(create_partition_cmd cmd, model::offset offset) {
 ss::future<std::error_code>
 topic_table::apply(move_partition_replicas_cmd cmd, model::offset o) {
     auto tp = _topics.find(model::topic_namespace_view(cmd.key));
-    if (tp == _topics.end() || !tp->second.is_topic_replicable()) {
```

Comment:
Should we use another error instead `topic not exists` for not replicable topic?